charts/dremio/templates/dremio-coordinator.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   name: dremio-coordinator
 spec:
   serviceName: "dremio-cluster-pod"
-  replicas: {{.Values.coordinator.count}}
```

Comment:
?

Reply:
My bad, I was testing it in for my branch